### ISSUE TITLE
Running Gradle in a non-Gradle project shouldn't create `.gradle` folder

### ISSUE THREAD

**Issue by lacasseio** [gradle/gradle#9533]
### Expected Behavior

No `.gradle` folder is created in non-Gradle project (e.g. any folder that is not part of a Gradle project or multi Gradle project).

### Current Behavior

Running `gradle help` in a folder outside a Gradle project will create a `.gradle` folder with caching data.

**Comment by dushechka** [comment]
@lacasseio It's not reproduced in 7.0 snapshot:

```
$ /usr/local/gradle-source-build/bin/gradle help

Welcome to Gradle 7.0-20210301084914+0000!

-//-

Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
Executing Gradle tasks as part of an undefined build is not supported. Make sure that you are executing Gradle from a folder within your Gradle project. Your project should have a 'settings.gradle(.kts)' file in the root folder.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 12s
```
```
$ ls -l | grep .gradle
$ 
```
Perhaps, this issue should be closed?

**Comment by shoaniki** [comment]
while it's true that this appears to be fixed in 7.0, i'm just gonna point out that `ls -l | grep .gradle` is by definition never going to see a .gradle directory

**Comment by dushechka** [comment]
> 
> 
> while it's true that this appears to be fixed in 7.0, i'm just gonna point out that `ls -l | grep .gradle` is by definition never going to see a .gradle directory

Sorry. But yeah, it's true. (:

**Comment by ljacomet** [comment]
Closing as obsolete, this issue and other problems when running Gradle in your user home have been fixed on the 7.x line.
